salt/grafana/init.sls

```diff
@@ -20,12 +20,22 @@
                          'PNDA Kafka Brokers.json',
                          'PNDA.json'] %}
 
+{% set haproxy_service = salt['pnda.generate_external_link']('haproxy',':8433') %}
+{% set grafana_config_dir = '/etc/grafana' %}
 
 grafana-server_pkg:
   pkg.installed:
     - sources:
       - grafana: {{ mirror_location + settings['package-source'] }}
 
+# register grafana as an internal service
+grafana-service_registration:
```

Comment:
This only runs when orchestration runs. Elsewhere we're tying this into systemd. One for discussion.